scripts/deploy/troubleshooting/get-all-logs

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function usage() {
+  echo "Usage: $0 <namespace>"
+  echo
+  echo "This script saves logs from all pods within the specified namespace into local files and then creates a tar.gz archive of these logs."
+  echo
+  echo "Arguments:"
+  echo "  namespace   - The namespace from which to retrieve and archive pod logs."
+  echo
+  echo "Example: $0 mynamespace"
+  exit 1
+}
+
+if [ "$#" -ne 1 ]; then
+  usage
+fi
+
+NAMESPACE=$1
+
+mkdir -p "$NAMESPACE-logs"
```

Comment:
see below about not creating and deleting a directory with a user input var.  As well, we can clean it up in case there's old logs in there from a different namespace or prior time the called this script on same namespace:

```suggestion
mkdir -p "/tmp/cht-eks-logs"
rm "/tmp/cht-eks-logs/*.log"
```

Reply:
consider using `mktemp` and trap, example:
https://github.com/medic/cht-core/blob/4ced6937a73b105fa2a970d729e3803afd29893a/haproxy-healthcheck/requirements/update.sh#L67-L69